package.json

```diff
@@ -46,12 +46,16 @@
     "classnames": "^2.3.2",
     "react-bootstrap-typeahead": "^6.1.2",
     "react-datepicker": "^4.11.0",
+    "react-jsx-runtime": "^1.0.0-alpha.1",
     "react-number-format": "^5.1.4",
     "rollup-plugin-import-css": "^3.2.1",
     "use-debounce": "^9.0.4"
   },
   "devDependencies": {
     "@babel/core": "^7.21.8",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
```

Comment:
This should be moved to the dependencies instead of the dev dependencies.

Reply:
Moved into dependencies